tools/isobuild/linker.js

```diff
@@ -145,32 +150,121 @@ _.extend(Module.prototype, {
     // Prologue
     var chunks = [];
 
+    const result = {
+      // This object will be populated with .source, .servePath,
+      // .sourceMap, and (optionally) .exportsName properties before being
+      // returned from this method in a singleton array.
+      servePath: self.combinedServePath,
+    };
+
     // Emit each file
-    _.each(self.files, function (file) {
-      if (!_.isEmpty(chunks)) {
-        chunks.push("\n\n\n\n\n\n");
+    if (self.useMeteorInstall) {
+      const tree = {};
```

Comment:
Can we put this clause in a separate function? This function is getting *very* long. (I know that this is definitely by far not the worst offender in tool code)